src/opm/output/eclipse/Summary.cpp

```diff
@@ -1790,13 +1791,15 @@ namespace Evaluator {
     }
 } // namespace Evaluator
 
-void reportUnsupportedKeywords(std::vector<std::string> keywords)
+void reportUnsupportedKeywords(std::vector<Opm::SummaryNode> keywords)
 {
     std::sort(keywords.begin(), keywords.end());
     auto uend = std::unique(keywords.begin(), keywords.end());
 
-    for (auto kw = keywords.begin(); kw != uend; ++kw)
-        ::Opm::OpmLog::info("Summary keyword '" + *kw + "' is unhandled");
+    for (auto node = keywords.begin(); node != uend; ++node) {
+        const auto& location = node->location();
+        ::Opm::OpmLog::warning("Unhandeled summary keyword '" + node->keyword() + "' at " + location.filename + ", line " + std::to_string(location.lineno));
```

Comment:
What if the location was defaulted? Should we not then have a message without the location info?

Also a typo: "Unhandeled" -> "Unhandled"

Reply:
OK - I have added a more explicit content for the auto created locations:
```
Lcoation rst_location("<Added automatically to support restart>", 0);
```

Have fixed the typo.

Reply:
Sounds good!